web-console/monitoring/alerts/alerts.md

```diff
@@ -0,0 +1,15 @@
+# Alerts
+
+## Alert Metrics
+
+![Alert details view with metrics](img/alert-metrics.png)
+- Initially the chart should be zoomed to a time period that at least includes the `for` attribute of the alert, but preferably 3-5 times that period as loading times allow
+- An orange line with an alert icon (if possible) shows the firing threshold for the alert
+- Clicking on the `View in Metrics` link takes the user to the metrics page (prometheus UI) and pre-populated with the relevant query for the alert
```

Comment:
Wondering if we can find a better link label for this.  I feel like _View in Prometheus_ seems more straightforward, but curious on other's thoughts

Reply:
I left that open in case prometheus ever gets replaced with a different UI for viewing metrics